src/lib/pipeline_configuration.py

```diff
@@ -57,15 +59,15 @@ def __init__(self, raw_field, coded_field, coda_filename, cleaner=None, code_sch
 class PipelineConfiguration(object):  
     DEV_MODE = True
 
-    PROJECT_START_DATE = isoparse("")
-    PROJECT_END_DATE = isoparse("")
+    PROJECT_START_DATE = isoparse("2019-02-04T00:00:00+0300") #To change before production time
+    PROJECT_END_DATE = isoparse("2019-05-14T00:00:00+0300")
 
     # Radio and follow up questions coding plans
-    RQA_FA_CODING_PLANS = [
+    RQA_FOLLOW_UP_CODING_PLANS = [
```

Comment:
Why are these called follow-up coding plans? Aren't these the main radio question flows?

(Previous projects that I have worked on have had rqa flows which handle the answers to each week's radio question, demographic profiling flows which run after we receive an rqa answer, and then follow-up surveys, which are a bit like demographics but ask follow-on questions about the radio shows, and are sent at set times after the rqa flows. I think I'm confused as to whether you're using "follow-up" in this usual AVF sense or in some new sense unique to this project)

Reply:
Your right, Its the same context as previous projects. Have combined RQA and Follow up coding plans in one list so that I can apply some of the autocode and apply manual code methods on them without having to rewrite the methods twice. Will give the coding plan a more clear name RQA_&_FOLLOW_UP_CODING_PLANS

Reply:
Is this definitely the right place to put the follow up surveys (the other option being in the same list as the demographics)?

I think it depends on where you want to put that data in the individuals analysis csv i.e. do you have a new row per answer like the RQAs, or fill in a new column for every RQA response, like we do with the demographics? It probably also depends on whether you're expecting one response to each follow-up survey or whether people can send repeat replies to the same question. What does RDA need for this project?

Reply:
Discussed offline.